src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/RemoveAzureStorageAccount.cs

```diff
@@ -44,6 +44,8 @@ public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
 
+            WriteWarning("Will add confirmation of executing the cmdlet and Force parameter in a future release. ");
```

Comment:
Here in this cmdlet, we'll add a Force parameter instead of removing one as we'll add ShouldProcess&ShouldContinue confirmation logic. What do you think about the warning message?